connectors/source.py

```diff
@@ -767,12 +780,14 @@ async def download_and_extract_file(
                 doc = await self.handle_file_content_extraction(
                     doc, source_filename, temp_filename
                 )
+            self.error_monitor.track_success()
```

Comment:
won't this end up double-counting? Since `sink.py` also uses the monitor for the lazy downloads?

Reply:
True, will fix!

Reply:
I don't really know what to do with that, we have several potential points of failure/success:

Extraction:
- File was successfully downloaded
- File was not successfully downloaded

Ingestion:
- File was successfully ingested
- File was not successfully ingested

It's possible to have all the permutations!

For example, file was not downloaded successfully, but we still choose to upload its metadata to Elasticsearch and everything worked out - is it a failure?

I'll open this up to Slack